### PR TITLE
1051 [SEO] Library Page Fix and CSS split by Block

### LINKED ISSUE #1051: [SEO] CSS split by Block - library

### PR DESCRIPTION
**Pull request by sharanagoudapatil**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #1051 

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/
- After: https://1051-css-split-library--danaher-ls-aem--hlxsites.hlx.page/us/en/library/antibodies
- After: https://1051-css-split-library--danaher-ls-aem--hlxsites.hlx.page/us/en/library/antibody-generation-hybridoma

### CODE REVIEW

File: templates/library/library-dev.css [hunk 1]
```diff
@@ -1,15 +0,0 @@
-.library main {
-  @apply mx-auto max-w-7xl flex flex-row gap-8 max-w-7xl mx-auto w-full bg-white;
-}
-
-.library main .section {
-  @apply px-6 xl:pl-0;
```
Comment (rgravitvl) [inline templates/library/library-dev.css:6]: Did you add this to JS ?

File: templates/library/library-dev.css [hunk 2]
```diff
@@ -1,15 +0,0 @@
...
-}
-
-.library main > .section:first-child {
-  @apply lg:w-recent-articles-left;
```
Comment (rgravitvl) [inline templates/library/library-dev.css:10]: Did you add this to JS ?

File: templates/library/library-dev.css [hunk 3]
```diff
@@ -1,15 +0,0 @@
...
-}
-
-.library main .section .call-to-action-wrapper h2 {
-  @apply !text-2xl p-0 m-0 tracking-tight sm:text-3xl text-white font-normal tracking-wide;
```
Comment (rgravitvl) [inline templates/library/library-dev.css:14]: Did you add this to JS ?

**Review by rgravitvl — CHANGES_REQUESTED** [review]

File: templates/library/library-dev.css (hunk 3 above)
Comment (sharanagoudapatil) [reply templates/library/library-dev.css:14]: This was added in call-toaction.js, so no need to add here again. So removed this.

File: templates/library/library-dev.css (hunk 2 above)
Comment (sharanagoudapatil) [reply templates/library/library-dev.css:10]: updated under 'columns-dev.css' as it there was same code added for .blog main like below,
image

**Review by davenichols-DHLS — APPROVED** [review]
Looking good in the front end.  @rgravitvl  - please can you confirm.

**Review by rgravitvl — APPROVED** [review]

**Pull request merged**